### PR TITLE
Symbols bug fixes and dev improvements

### PR DESCRIPTION
**Pull request by chrismwendt**
- I forgot to list languages other than TypeScript on the command line in https://github.com/sourcegraph/sourcegraph/pull/2139
- Updates the ctags version
- Fixed some broken regexes
- Automatically rebuilds the symbols image upon modification of any `cmd/symbols/**` file

### CODE REVIEW

**Comment by sqs** [comment]
If you omitted Scala (only included TS), how was I able to get Scala j2d in the gif I posted on the other PR?

**Comment by chrismwendt** [comment]
That must have come from the basic-code-intel package in sourcegraph-scala and the definition pattern https://github.com/sourcegraph/sourcegraph-scala/blob/62b492aa0b9a3ca8ebf54b5fb0e414e6a659490a/src/extension.ts#L5

File: cmd/server/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.9 AS ctags
 
 # hadolint ignore=DL3003,DL3018,DL4006
-RUN apk --no-cache add --virtual build-deps curl jansson-dev libseccomp-dev linux-headers autoconf pkgconfig make automake gcc g++ binutils && curl https://codeload.github.com/universal-ctags/ctags/tar.gz/7918d19fe358fae9bad1c264c4f5dc2dcde5cece | tar xz -C /tmp && cd /tmp/ctags-7918d19fe358fae9bad1c264c4f5dc2dcde5cece && ./autogen.sh && LDFLAGS=-static ./configure --program-prefix=universal- --enable-json --enable-seccomp && make -j8 && make install && cd && rm -rf /tmp/ctags-7918d19fe358fae9bad1c264c4f5dc2dcde5cece && apk --no-cache --purge del build-deps
+RUN apk --no-cache add --virtual build-deps curl jansson-dev libseccomp-dev linux-headers autoconf pkgconfig make automake gcc g++ binutils && curl https://codeload.github.com/universal-ctags/ctags/tar.gz/08a69b5a030cf56c079e3006238b992b34c9cc51 | tar xz -C /tmp && cd /tmp/ctags-08a69b5a030cf56c079e3006238b992b34c9cc51 && ./autogen.sh && LDFLAGS=-static ./configure --program-prefix=universal- --enable-json --enable-seccomp && make -j8 && make install && cd && rm -rf /tmp/ctags-08a69b5a030cf56c079e3006238b992b34c9cc51 && apk --no-cache --purge del build-deps
```
Comment (keegancsmith) [inline cmd/server/Dockerfile]: You need to copy in the .ctags.d files from symbols somewhere.

**Comment by chrismwendt** [comment]
Thanks @keegancsmith, I have since fixed those issues.

**Comment by nicksnyder** [comment]
CHANGELOG!

**Comment by chrismwendt** [comment]
Added to the CHANGELOG in https://github.com/sourcegraph/sourcegraph/commit/fecc76a9ed093406f2b9e50672adb93896092dcc

**Pull request merged**